publish-to-verdaccio.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/sh -e
```

Comment:
I would move this to the `scripts` or `helpers` package or add some comments about what this should be used for.

Reply:
Moved.